var/spack/repos/builtin/packages/py-openpmd-viewer/package.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyOpenpmdViewer(PythonPackage):
+    """Python visualization tools for openPMD files"""
+
+    homepage = "https://www.openPMD.org"
+    url      = "https://github.com/openPMD/openPMD-viewer/archive/refs/tags/1.2.0.tar.gz"
+    git      = "https://github.com/openPMD/openPMD-viewer.git"
+
+    maintainers = ['RemiLehe', 'ax3l']
+
+    version('1.2.0', sha256='a27f8ac522c4c76fd774095e156a8b280c9211128f50aa07f16ac70d8222384d')
+
+    variant('jupyter', default=False,
+            description='Enable Jupyter Widget GUI')
+    variant('numba', default=False,
+            description='Enable accelerated depositions for histograms')
```

Comment:
Usually variants in Python packages match "extras_require" in the `setup.py`, but these seem to be custom.

Reply:
Yes, forgotten in the repo. Fix in: https://github.com/openPMD/openPMD-viewer/pull/327